monkey/infection_monkey/readme.txt

```diff
@@ -46,9 +46,11 @@ Tested on Ubuntu 16.04 and 17.04.
 2.	Build Sambacry binaries
 	a. Build/Download according to sections at the end of this readme.
 	b. Place the binaries under [code location]\infection_monkey\bin, under the names 'sc_monkey_runner32.so', 'sc_monkey_runner64.so'
+	c. Run command chmod +x to make binaries executable.
```

Comment:
SOs don't need to be marked as executable

Reply:
They will be executed only on the target machine not on the local machine

Reply:
These instructions are for **building** monkey binaries. @amanjiofficial you ran into problems trying to **build** monkey binaries or trying to **run** monkey?